cli/code_scanner.py

```diff
@@ -328,19 +340,26 @@ def handle_scan_result(context, scan_result, command_scan_type, scan_type, sever
 def perform_pre_scan_documents_actions(context: click.Context, scan_type: str, documents_to_scan: List[Document],
                                        is_git_diff: bool = False):
     if scan_type == SCA_SCAN_TYPE:
+        logger.info(
```

Comment:
debug

Reply:
I put it on info since I would like to update user on processing

Reply:
Fixed